### PR TITLE
Incorporated feedback from #23

### PR DESCRIPTION
**Pull request by gruetter**
Thanks to @guywmartin and @rrrutledge for your feedback on the TC article. I have incorporated almost all of it, but there are still two sections which contain my personal experience and some personal examples. I'll highlight those sections with a remark. We need to decide whether or not we want to keep them (I'm fine with removing them, as they are in the video).

### CODE REVIEW

**Comment by gruetter** [comment]
@rrrutledge @guywmartin : If you have time, could you pls. respond to the two open questions above?

**Comment by rrrutledge** [comment]
Sounds great, I'm just not clear which two questions you mean?

**Comment by gruetter** [comment]
I was referring to the question whether or not to remove the personal examples from the article altogether. I realise I've already done that with 246bfc8, though. So we're good to merge, IMO, @rrrutledge .

**Comment by rrrutledge** [comment]
Oh, I accidentally committed some updates directly to the branch.  I meant to commit to a fork so that I could make a PR to the branch under review here.  Hopefully those edits look OK.  If not we can revert.

File: trusted-committer-article.md [hunk 1]
```diff
@@ -490,21 +465,22 @@ contribution to the community and accelerate your own growth.
...
-the importance of having capable TCs for the success of any InnerSource 
-initiative and the level of empowerment that this role requires.
+to making contributions, upleveling the community and advocating the
+community's needs in their organization. We also talked about how to become a
+TC and what it takes to fill that role. Working as a TC will be demanding but
+it will also be very fulfilling and will help you amplify your value contribution
+in your company.
+
+In that sense, we hope that this video inspired you to set off on a path
+towards becoming a TC as well. And if you're doubt, why not just try it out?
```
Comment (rrrutledge) [inline trusted-committer-article.md]: > And if you're doubt, why not just try it out?

I think that this line works out well in the spoken text, but feels a little off-tone here in writing.  Recommend just removing it.

**Comment by gruetter** [comment]
> I think that this line works out well in the spoken text, but feels a little off-tone here in writing. Recommend just removing it.

Agreed, @rrrutledge . What do you think: should we also remove the reference to the _video_ in line 474?

**Comment by rrrutledge** [comment]
> should we also remove the reference to the video

Agree.  For expediency just went and made the changes.  Feel free to revert or if it looks good to you and @guywmartin we can merge.

**Comment by gruetter** [comment]
Ok great, @rrrutledge and @guywmartin . Thanks a lot for the review.

**Pull request merged**